XPlat.ApplicationModel.DataTransfer/Clipboard.Windows.cs

```diff
@@ -0,0 +1,51 @@
+#if WINDOWS_UWP
+namespace XPlat.ApplicationModel.DataTransfer
+{
+    using System;
+    using System.Threading.Tasks;
+
+    /// <summary>Gets and sets information from the clipboard object.</summary>
+    public static class Clipboard
+    {
+        public static event EventHandler<object> ContentChanged;
```

Comment:
This event is missing XML docs.

Reply:
Also updated with the relevant summary tag here